app/models/transfer-store/transfer-store.ts

```diff
@@ -20,7 +20,7 @@ export const TransferStoreModel = TxStoreModel
   }))
   .views(self => ({
     get receiverAddress() {
-      return self.liker ? self.liker.cosmosWallet : self.target
+      return self.liker ? self.liker.likeWallet : self.target
```

Comment:
Would this PR release before or after chain upgrade? We do not support like prefix before upgrade is complete

Reply:
1. keep cosmosWallet for now, and release another version after the chain upgrade
2. Config which addresses to use with remote config

Reply:
both would work, but for 2 the chain id won't change, so maybe need a prefix field

Reply:
Added a remote config `CURRENT_PREFIX": "like`,  now can switch this config to return a different address.